docs/components/pagination.md

```diff
@@ -43,11 +43,18 @@ Simple pagination inspired by Rdio, great for apps and search results. The large
 
 Links are customizable for different circumstances. Use `.disabled` for unclickable links and `.active` to indicate the current page.
 
+{% callout warning %}
+#### Disabled links and keyboard access
+
+Although links styled as `.disabled` prevent mouse/touch click functionality, they will still be focusable with the keyboard. To prevent this, include `tabindex="-1"` on the link, or use a static, non-focusable element instead.
```

Comment:
Given that we already say that  `pointer-events: none` isn't reliable with regard to browser compatibility, and might be removing it, I wouldn't say that they "prevent mouse/touch click functionality".